ecs-cli/modules/clients/aws/cloudformation/template.go

```diff
@@ -163,9 +163,14 @@ var template = `
     },
     "CreateSecurityGroup": {
       "Fn::Equals": [
-        {
-          "Ref": "SecurityGroup"
-        },
+				{
+					"Fn::Join": [
+						"",
+						{
+							"Ref": "SecurityGroup"
```

Comment:
So CreateSecurityGroup is actually just a test to tell us if we need to create one or not. If I had named it, I would have called it something like "NeedToCreateSecurityGroup" or "IsSecurityGroupSpecified"- because that is what it actually does. It returns true if no group was specified. A join is needed because the parameter SecurityGroup is now a list, but Fn::Equals needs its arguments as strings. So the join converts the list to a string.